examples/donation-balancesOf/index.mjs

```diff
@@ -0,0 +1,74 @@
+import {loadStdlib} from '@reach-sh/stdlib';
+import * as backend from './build/index.main.mjs';
+import launchToken from '@reach-sh/stdlib/launchToken.mjs';
+
+const stdlib = loadStdlib(process.env);
+
+(async () => {
```

Comment:
I know all the examples do this, but it isn't actually needed and we should get rid of it